lib/core/connection_factory/http.rb

```diff
@@ -5,7 +5,9 @@
 module Core
   module ConnectionFactory
     class Http
-      def self.build(url, timeout: 5, open_timeout: 5, retries: 2)
+      TIMEOUT = ENV.fetch('FRONTEND_HTTP_REQUEST_TIMEOUT').to_i
```

Comment:
And I happy because if you put 'banana' or similar then the timeout tests will fail. Exactly what I expect too.